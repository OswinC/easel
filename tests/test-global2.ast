function int foo(int x)
{
a = x + 1;
}

int a;
foo(73);
print(a);
