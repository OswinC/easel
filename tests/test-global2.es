int a;
bool b;
pix p;

print(a);
print(b);
print(p);
