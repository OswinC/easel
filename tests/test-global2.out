255
#ff000000
#ffffff00
#ff000000
